### PR TITLE
🔐 Secure admin site with proxy and SSL

### LINKED ISSUE #34: 🔐 Secure admin site with SSL
### Because

All traffic to the `ov-admin` ingress needs to be secured with `https`

### Done when

- [x] All traffic to `ov-admin` is SSL encrypted

### PR DESCRIPTION
**Pull request by mrharpo**
# Admin proxy
## `ov-nginx`
- Adds ingress control for admin site: ov-admin.k8s.wgbhdigital.org
- `proxy_pass` to `ov-wag`
- Adds `server_name` to frontend config: ovfrontend.k8s.wgbhdigital.org

Related to [ov-wag #48](https://github.com/WGBH-MLA/ov-wag/issues/48)
Also: Closes #34

### CODE REVIEW

File: ov-nginx/nginx.conf [hunk 1]
```diff
@@ -3,8 +3,20 @@ events {
...
+    server_name ov-admin.k8s.wgbhdigital.org;
+    location / {
+      proxy_pass http://ov-wag;
+    }
+  }
+
+  # Frontend
   server {
     listen 80 default_server;
+    server_name ovfrontend.k8s.wgbhdigital.org;
```
Comment (afred) [inline ov-nginx/nginx.conf:19]: ov-frontend vs ovfrontend?

Comment (mrharpo) [reply ov-nginx/nginx.conf:19]: Came configured that way. 

`ov-front`?

Comment (afred) [reply ov-nginx/nginx.conf:19]: ah, i see. i guess we can change it later if we want, but shouldn't block this.

**Review by afred — APPROVED** [review]
👍

**Pull request merged**